### PR TITLE
Add post

### PR DESCRIPTION
**Pull request by PJSalter**
- fixed issue with package.json by adding The "-r dotenv/config" bit that tells the dotenv library to read the .env file and pass es all the values inside it to your application. Then  could access them in your JS code with process.env.VAR_NAME.
- added this in the "dev".
- adding a POST with post function, containing a request and response.
- this then updates the database. The form submits a POST request to the same path. The post handler logs whatever data was submitted.
- using the INSERT INTO SQL command to create a new user based on the user-submitted information. This also contains all four values. such as $1, $2, $3, $4 which are there so to speak placeholders.
- this will then query the database.
- Querying for data and set up of connection.js which me and Sonia pair programmed on, this sent sql commands to the database, checking that this runs the databases URL within the .env file itself.
- if it doesn't then it'll throw and error.

Hope I've made sense on the tasks completed 😸 

Let me know what you think 🤩 ✨

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "cypress run",
     "start": "node server.js",
-    "dev": "nodemon server.js"
+    "dev": "nodemon -r dotenv/config server.js"
```
Comment (mariaalouisaa) [inline package.json:9]: I can't believe this is what we were missing!! Good spot 🥳🥳🥳

File: routes/home.js [hunk 2]
```diff
@@ -49,5 +51,26 @@ function get(request, response) {
...
 }
 
+// function post(req, res) {
+//   console.log(req.body);
+//   return null;
+// }
+
+function post(request, response) {
+  const insert_user = /*sql*/ `
+      INSERT INTO users(username, age, base_flavour, fandom) VALUES($1, $2, $3, $4)
```
Comment (mariaalouisaa) [inline routes/home.js:61]: This looks super tidy and easy to read. Thank you for finishing this function off.

File: routes/home.js [hunk 3]
```diff
@@ -49,5 +51,26 @@ function get(request, response) {
...
+  ];
+
+  db.query(insert_user, values).then(() => {
+    response.redirect("/");
+  });
+}
+
 // exporting for server.js
-module.exports = { get };
+module.exports = { get, post };
```
Comment (mariaalouisaa) [inline routes/home.js:76]: Lovely modularizing 👌

**Review by mariaalouisaa — COMMENTED** [review]
I am happy for this to be merged, looks great!!! Thanks for doing this "out of office hours"

File: routes/home.js (hunk 2 above)
Comment (PJSalter) [reply routes/home.js:61]: Thank you ✨

File: routes/home.js (hunk 3 above)
Comment (PJSalter) [reply routes/home.js:76]: 😸 ⭐

**Comment by lascellesabercrombie** [comment]
Looks good to me! Thanks, Peter. Is there a way of testing if the post function is working with the database until we either add Sonia's display functionality and/or merge this to main and push it to Heroku? In any case, I'm happy for this to be merged.

**Pull request merged**